### PR TITLE
Packaging and adding ROS exmaple 

### PR DESCRIPTION
**Pull request by ibraEssam**
Hi, 
I've created a package for easier use and distribution. 
now you can just 
`from pycarmaker import CarMaker, Quantity`
also, I've added a new ROS node example and updated README.
it's gonna be even better if you uploaded it to [PyPi](https://packaging.python.org/tutorials/packaging-projects/)

### CODE REVIEW

File: examples/cm_ros_node.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
```
Comment (gmnvh) [inline examples/cm_ros_node.py:1]: What if we call this file Ex03_CarMaker_Ros_Bridge.py?

**Review by gmnvh — CHANGES_REQUESTED** [review]
Let's change the ROS example file to Ex03_CarMaker_ROS_Bridge.py. What do you think?

**Comment by gmnvh** [comment]
@HemaZ Awesome work here! Thanks for your contribution.

File: examples/cm_ros_node.py (hunk 1 above)
Comment (ibraEssam) [reply examples/cm_ros_node.py:1]: yes sure feel free to make any changes

**Pull request merged**